test/integration/cli-domain-package-static-files-minify-file.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var expect = require('chai').expect;
+
+describe('cli : domain : package-static-files : minify-file', function(){
+	var minifyFile = require('../../src/cli/domain/package-static-files/minify-file');
+
+	describe('when minifying .js file', function(){
+
+		describe('when file contains es6', function(){
+
+			var content = 'const hi = (name) => `hello ${name}`;';
+			var minified;
+
+			before(function(){
+				minified = minifyFile('.js', content);
+			});
+
+			it('should minify it', function(){
```

Comment:
I personally think we should not test how the internal minification library works; that's why I was proposing a snapshot test. You will end up having a file containing your uglified code in one place. which is more or less what you do have now. What about a pairing session w/ you after this PR has been merged?

// cc @matteofigus @nickbalestra